CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,55 @@
+# Fabric Code of Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, culture, age, religion, nationality, or other similar characteristic.
+
+## Examples of behavior that contributes to creating a positive environment include:
+
+* Be kind and courteous to others
```

Comment:
"Examples of behavior" implies nouns, hence `Being` being more appropriate here.

Reply:
Good catch, that was a typo on my part. Thanks!